tromega/lib/views/in_training/track_training_view.dart

```diff
@@ -10,36 +10,47 @@ import '../../widgets/tracking/displays/execution_page.dart';
 import '../../widgets/tracking/interactives/exercise_thumbnail.dart';
 
 class TrackingView extends StatefulWidget {
-  const TrackingView({Key? key, required this.trainingDayId}) : super(key: key);
+  const TrackingView(
+      {Key? key, required this.trainingDayId, required this.trainingPlanId})
+      : super(key: key);
   final String trainingDayId;
+  final String trainingPlanId;
   @override
   State<TrackingView> createState() => _TrackingViewState();
 }
 
-class _TrackingViewState extends State<TrackingView> {
+class _TrackingViewState extends State<TrackingView>
+    with TickerProviderStateMixin {
   late TrainingSession lastSession;
   late TrainingSession thisSession;
   late TrackingHttpHelper trackingHttpHelper;
   late CustomTimerController _timerController;
+  late int timerSeconds;
   late String trainingDayId;
+  late String trainingPlanId;
   bool trainingFinished = false;
   bool fetching = true;
   int highlightedPage = 0;
 
-
   final PageController _pageController = PageController(initialPage: 0);
 
   @override
   initState() {
     /// hardcoded for debugging purposes
     trainingDayId = widget.trainingDayId;
+    trainingPlanId = widget.trainingPlanId;
     trackingHttpHelper = const TrackingHttpHelper();
-    _timerController = CustomTimerController();
+    timerSeconds = 180;
```

Comment:
man kann das echt nicht ändern mit den 180s?

Reply:
Kann in jedem Training vom Nutzer angepasst werden, aber leider nicht gespeichert. Datamodell müsste dafür angepasst werden